### PR TITLE
NI-DCPower: Update create advanced sequence docs, metadata

### PR DESCRIPTION
**Pull request by olsl21**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
~- [ ] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~
~- [ ] I've added tests applicable for this pull request~

### What does this Pull Request accomplish?

- Update NI-DCPower metadata (to 22.8.0d129) and functions_addon.py to update the docs for `_create_advanced_sequence_with_channels()` and `create_advanced_sequence()` to include:
  - PXIe-4147 and PXIe-4190
  - New LCR-related attributes

### What testing has been done?

The updated docs are as expected.
image

### CODE REVIEW

File: src/nidcpower/metadata/functions.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-DCPower API metadata version 22.8.0d88
+# This file is generated from NI-DCPower API metadata version 22.8.0d9999
```
Comment (olsl21) [inline src/nidcpower/metadata/functions.py:2]: Awaiting official metadata export

**Review by sbethur — APPROVED** [review]

File: src/nidcpower/metadata/attributes.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-DCPower API metadata version 22.8.0d88
```
Comment (marcoskirsch) [inline src/nidcpower/metadata/attributes.py:2]: await official metadata build

**Review by marcoskirsch — CHANGES_REQUESTED** [review]

File: src/nidcpower/metadata/attributes.py (hunk 2 above)
Comment (olsl21) [reply src/nidcpower/metadata/attributes.py:2]: I have updated the metadata version to 22.8.0d129 in my latest commit

File: src/nidcpower/metadata/functions.py (hunk 1 above)
Comment (olsl21) [reply src/nidcpower/metadata/functions.py]: I have updated the metadata version to 22.8.0d129 in my latest commit

**Review by marcoskirsch — APPROVED** [review]

**Pull request merged**